### DISCUSSION TITLE
无法加载编译后的插件

### DISCUSSION THREAD

**Discussion by kaixinol** [the1812/Bilibili-Evolved#4601] [提问 / Q&A] [answered]
我此前没有接触过油猴脚本开发和前端，想自力更生修改匹配规则，今天尝试按照贡献指南修改了 https://github.com/the1812/Bilibili-Evolved/blob/e7dba14fc8f3d8f1a222088d0b48e1b5cebd94df/registry/lib/components/utils/image-exporter/feed.ts#L24 
把其中的css选择器改成了：
```diff
-.main-content .img-content, .bili-album__preview__picture__img
+.main-content .img-content, .bili-album__preview__picture__img, div.bili-album img
```
然后运行`pnpm run build-features`编译，生成了`/media/askirin/Data/Project/Bilibili-Evolved/registry/dist/components/utils/image-exporter.js`
安装本地插件后，发现并没有按预期添加选项
![image](https://github.com/the1812/Bilibili-Evolved/assets/100816732/28d744cd-9516-4655-8bab-38ac2b8bb122)

**Comment by the1812** [comment, marked as answer]
按照贡献指南的话应该是这里的步骤? 应该不会用到 `build-features` 吧
![Snipaste_2024-01-14_15-54-26](https://github.com/the1812/Bilibili-Evolved/assets/26504152/f182ebc6-3fe9-4733-ac16-d8da08372701)

**Reply by kaixinol** [reply]
明白了，之前看文章时没看懂，没搜到dev-server是什么东西😂
